packages/backend/src/graphql/resolvers/liquidity.test.ts

```diff
@@ -502,7 +502,7 @@ describe('Liquidity Resolvers', (): void => {
               peerId: peer.id,
               amount: startingBalance.toString(),
               idempotencyKey: uuid(),
-              timeout: timeoutOnePhase
+              timeoutSeconds: timeoutOnePhase
```

Comment:
I think we can get rid of `timeoutOnePhase` variable, and just replace them all with 0s (I think that reads easier instead of having to know what `timeoutOnePhase` is)

Reply:
Ok. done. :-)